app/models/license.rb

```diff
@@ -33,23 +33,44 @@ class License
     'none' => 'No License'
   }.freeze
 
+  # valid, displayable (not selectable) licenses --- used to allow migration of legacy system data
+  DISPLAYABLE_LICENSES = {
+    'CC-BY-3.0' => 'CC Attribution 3.0 (Unsupported)',
+    'CC-BY-SA-3.0' => 'CC Attribution Share Alike 3.0 (Unsupported)',
+    'CC-BY-ND-3.0' => 'CC Attribution No Derivatives 3.0 (Unsupported)',
+    'CC-BY-NC-3.0' => 'CC Attribution Non Commercial 3.0 (Unsupported)',
+    'CC-BY-NC-SA-3.0' => 'CC Attribution Non Commercial Share Alike 3.0 (Unsupported)',
+    'CC-BY-NC-ND-3.0' => 'CC Attribution Non Commercial No Derivatives 3.0 (Unsupported)'
+  }.freeze
+
   # used for validation in work model
   sig { returns(T::Array[String]) }
   def self.license_list
-    ID_LABEL_HASH.keys
+    SELECTABLE_LICENSES.keys
   end
 
   # list for the work form pulldown
-  sig { returns(T::Array[T::Array[T.any(String, T::Array[String])]]) }
-  def self.grouped_options
-    GROUPINGS.map do |group|
+  sig { params(license: T.nilable(String)).returns(T::Array[T::Array[T.any(String, T::Array[String])]]) }
+  def self.grouped_options(license = nil)
+    options = GROUPINGS.map do |group|
       [group.fetch(:label), group.fetch(:options).map { |license_id| [label_for(license_id), license_id] }]
     end
+    return options unless license.in?(DISPLAYABLE_LICENSES)
+
+    # If the current license is merely displayable, add the displayable licenses to the dropdown as disabled options
+    options.append(
```

Comment:
I don't think we want to add this here, this would be in the selectable list. We want the user to select a new valid license if they make a change.

Reply:
@jcoyne It shouldn't because of the check on line 58. This only appends displayable licenses if one of them is already selected.

Reply:
👍